### PR TITLE
Add Policy Sentry by Salesforce

### PR DESCRIPTION
**Pull request by kmcquade**
## Review the Contributing Guidelines

Before submitting a pull request, verify it meets all requirements in the [Contributing Guidelines](https://github.com/donnemartin/awesome-aws/blob/master/CONTRIBUTING.md).

## Describe Why This Is Awesome

* Why is this awesome?

It generates secure least-privilege IAM Policies automagically. Writing complex IAM Policies can take place over about 30 seconds instead of hours in the IAM Visual editor / Policy Generator. 

--

Like this pull request?  Vote for it by adding a :+1:

### CODE REVIEW

**Comment by kmcquade** [comment]
@donnemartin friendly reminder :)

File: README.md [hunk 1]
```diff
@@ -935,6 +935,7 @@ Community Repos:
 * [Netflix/bless :fire::fire::fire::fire::fire:](https://github.com/Netflix/bless) - SSH Certificate Authority that runs as a Lambda function.
 * [Netflix/security_monkey :fire::fire::fire::fire::fire:](https://github.com/Netflix/security_monkey) - Monitors policy changes and alerts on insecure configurations.
 * [RiotGames/cloud-inquisitor :fire::fire:](https://github.com/RiotGames/cloud-inquisitor) - Tool to enforce ownership and data security.
+* [salesforce/policy_sentry :fire::fire::fire:] - Generate secure, least privilege IAM Policies automagically
```
Comment (donnemartin) [inline README.md:938]: @kmcquade could you add the link here and refresh the description based on current repo description?

* [salesforce/policy_sentry :fire::fire::fire:](https://github.com/salesforce/policy_sentry) - IAM Least Privilege Policy Generator.

Comment (kmcquade) [reply README.md:938]: @donnemartin for some reason I'm not able to edit this PR. I think I must have deleted my fork. I'm going to close this one and then make a new one. Cool?

Comment (donnemartin) [reply README.md:938]: Sounds good!

**Comment by kmcquade** [comment]
Closing in favor of #197 

FYI @donnemartin :)

**Pull request closed without merging**